educator/src/Dscp/Educator/Web/Server.hs

```diff
@@ -1,49 +1,58 @@
+{-# LANGUAGE TypeOperators #-}
+
 -- | Functions to serve Student HTTP API
 
 module Dscp.Educator.Web.Server
        ( serveStudentAPIReal
-       , convertStudentApiHandler
        ) where
 
 import Fmt ((+|), (|+))
 import Loot.Log (logInfo)
-import Servant (Handler, Server, hoistServer, serve, throwError)
+import Servant ((:<|>) (..), (:>), Handler, Server, hoistServer, serve)
 import Servant.Generic (toServant)
 
 import Dscp.Educator.Config (HasEducatorConfig)
-import Dscp.Educator.Launcher (EducatorContext, EducatorRealMode, EducatorWorkMode)
-import Dscp.Educator.Web.Bot.Handlers (addBotHandlers)
+import Dscp.Educator.Launcher (EducatorRealMode, EducatorWorkMode)
+import Dscp.Educator.Web.Bot (addBotHandlers)
+import Dscp.Educator.Web.Educator (EducatorAPI, convertEducatorApiHandler, educatorAPI,
+                                   educatorApiHandlers)
 import Dscp.Educator.Web.Params (EducatorWebParams (..))
-import Dscp.Educator.Web.Student.API (StudentAPI, studentAPI)
-import Dscp.Educator.Web.Student.Error (toServantErr, unexpectedToServantErr)
-import Dscp.Educator.Web.Student.Handlers (studentApiHandlers)
-import Dscp.Launcher.Rio (runRIO)
+import Dscp.Educator.Web.Student (StudentAPI, convertStudentApiHandler, studentAPI,
+                                  studentApiHandlers)
 import Dscp.Web (ServerParams (..), serveWeb)
 
-studentAPIServer
+type EducatorWebAPI = "api" :> (
+    "educator" :> EducatorAPI
+    :<|>
+    "student" :> StudentAPI
```

Comment:
No `"common"`?

Reply:
Waiting for @volhovm to create some ticket about it

Reply:
Let it be separate. It doesn't matter that much.